elastic-agent-client.proto

```diff
@@ -292,12 +292,28 @@ message CheckinAgentInfo {
     bool snapshot = 3;
 }
 
+// Feature flags configurations.
+// Added on Elastic Agent v8.7.1.
+message Features {
+    FQDNFeature fqdn = 2;
```

Comment:
I would also add the `Source` field just like the UnitExpectedConfig does. That will allow future features to be added and used without requiring the client to be modified and updated.